CHANGES.rst

```diff
@@ -312,6 +312,10 @@ astropy.tests
 
 - Fixed issue running doctests with pytest>=3.2. [#6423]
 
+- In anticipation of supporting test runs directly from ``pytest``, fixed
```

Comment:
The issue with this changelog entry is that it means that we are definitely committing to supporting running tests directly from pytest. Can you write the changelog entry in a way that doesn't imply this?